pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -314,10 +315,15 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 	}
 
 	// Hook container up with our bridge
-	res, err := plugin.addContainerToNetwork(plugin.netConfig, network.DefaultInterfaceName, namespace, name, id)
+	resT, err := plugin.addContainerToNetwork(plugin.netConfig, network.DefaultInterfaceName, namespace, name, id)
 	if err != nil {
 		return err
 	}
+	// Coerce the CNI result version
+	res, err := cnitypes020.GetResult(resT)
```

Comment:
Correct - this converts the result in to the 0.2.0 format, which is what kubenet already expects.